arbor/memory/allocator.hpp

```diff
@@ -173,9 +173,9 @@ class allocator : public Policy {
     using rebind = allocator<U, Policy>;
 
 public:
-    allocator() {}
-    ~allocator() {}
-    allocator(allocator const&) {}
+    allocator() = default;
+    ~allocator() = default;
+    allocator(allocator const&) = default;
```

Comment:
That's different behavior, no?

Reply:
The copy constructor might be arguable, however, `allocator has no members, so we are good?! Or am I missing something?